codalab/apps/authenz/views.py

```diff
@@ -98,7 +98,8 @@ def _translate_names(names):
         if names is None or type(names) is not list or len(names) == 0:
             return {'code': 400}
         for name in names:
-            if type(name) not in (str, unicode) or len(name) < 1:
+            # TODO: Ensure this check is correct
```

Comment:
When is this going to be confirmed? Can you go through all todos?

Reply:
Went through all to-dos. For some of the 2to3 changes where it cast to a list, I left it because it was a separate object and the explicit cast seemed to make sense.